views/EventView.js

```diff
@@ -148,11 +150,12 @@ export default class EventView extends React.Component {
                     <View style={Styles.inputRow}>
                         <Text style={Styles.inputTitle}>Time</Text>
                         <RadioGroup style={Styles.datetimeContainer}
+                            key = {this.state.date}
                             radioButtons = {this._generateRadioButtons()}
                             onPress = {buttons => {
                                 let selected = buttons.find(b => b.checked);
 
-                                if (selected.value === 0 && !this.state.customTime) {
+                                if (selected.value === "c" && !this.state.customTime) {
```

Comment:
This is going to reintroduce the "Clicking Custom Twice" error:
After clicking Custom once, customTime will be set to true.
Clicking Custom a second time will result in the else clause being executed since customTime is true, and splits will attempt to split the label "Custom" with the "-" separator.  Using splits[1] when defining newState will then result in an error for referencing a null value.

Reply:
I've yet to merge master into this branch, hence why that line reintroduces that error. When I do that, I'll make sure to keep an eye out for that!

Reply:
Actually, I just rechecked the code in master, and I accidentally introduced a major error in onPress by reordering the conditional branches when I fixed the "Clicking Custom Twice" error (newState never gets defined when clicking a button for a default time).
The code here is good except for the " && !this.state.customTime" condition, since it forces the execution of the else clause when it shouldn't.

So TL;DR: Don't use the code for onPress currently in master; use this code after removing " && !this.state.customTime"

Unless you're currently working on something in this branch, I can go ahead and do that if you want.

Reply:
My bad, I was at work @jzb0128.
I've now merged in the changes from master, as well as updating the code a bit for the time input. I looked at where you mentioned an error, and it looked like it wouldn't set the start/end time if the user selected one of the default times, which is definitely not what we'd want! Feel free to take a look at it yourself to see if it would still cause the issue with clicking on custom twice, but I don't think that it should.